application/listeners/posthelpers.go

```diff
@@ -257,6 +265,8 @@ func join(s1, s2 string) string {
 	return s1 + "-" + s2
 }
 
+var getT = notifications.GetEmailTemplate
```

Comment:
stylistic nit: I'd prefer to see this near the point of use or at the top of the file.

Reply:
I must have been calling it more than once in previous iteration before I finalized the code here.